contrib/gardener/scripts/deploy-karydia-cp

```diff
@@ -222,7 +222,8 @@ spec:
       automountServiceAccountToken: false
       containers:
       - name: karydia
-        image: karydia/karydia:0.1.0
+        # TODO: Add version number
+        image: eu.gcr.io/gardener-project/karydia/karydia
         imagePullPolicy: IfNotPresent
```

Comment:
If you'll change this `imagePullPolicy: IfNotPresent` to `imagePullPolicy: Always` as well, we're good to go. 😎

Reply:
Done